Warderobe/run.py

```diff
@@ -0,0 +1,74 @@
+import json
+import time
+import requests
+import Warderobe.LedController as led
+
+global_url = "http://localhost:8000"
+
+
+def login_warderobe(user_mail: str, serial_number: str):
+    url = global_url + "/warderobe/connect"
+    w_data = {"mail": user_mail,
+              "password": serial_number,
+              }
+    response = requests.post(
+        url, data=json.dumps(w_data), headers={"Content-Type": "application/json"}
+    )
+    return response
+
+
+def download_all_clothes(token: str):
+    url = global_url + "/collection/items/"
+    headers = {
+        "Authorization": "Bearer " + token,
+    }
+    response = requests.get(url, headers=headers)
+    return response
+
+
+def show_items_list(token: str):
+    url = global_url + "/warderobe/items"
+    headers = {
+        "Authorization": "Bearer " + token,
+    }
+    response = requests.get(url, headers=headers, timeout=60)
+    return response
+
+
+def parse_all_clothes(json_data):
+    parsed_data = []
+    for item in json_data:
+        item_data = item["Item"]
+        parsed_item = {
```

Comment:
shelf i position ma być później przechowywane w bazie danych aplikacji? Jeśli tak to trzeba dodać te pola w kodzie i zmodyfikować książkę, dając najlepiej znać Zuzi

Reply:
Nie to będą wewnętrzne danye szafy. Będzie to schowane w csv w szafie